lib/execution_engine2/constants.py

```diff
@@ -0,0 +1,3 @@
+""" Global EE2 constants. """
+ADMIN_READ_ROLE = "EE2_ADMIN_RO"
```

Comment:
We have a EE2Constants.py file already. Maybe we should use that?

Reply:
Good call, fixed